scout/parse/variant/callers.py

```diff
@@ -28,36 +36,51 @@ def parse_callers(variant, category="snv"):
     svdb_origin = variant.INFO.get("svdb_origin")
     raw_info = variant.INFO.get("set")
 
+    filter_status_default = "Pass"
+    if variant.FILTER is not None:
+        filter_status_default = "Filtered - {}".format(variant.FILTER.replace(";", " - "))
```

Comment:
This is one of the crucial changes. As noted below at its origin, the previous version would never add the actual FILTER tag on parsing. Python lazy initiation error. That would just have been a compile error in another language.